@here/harp-test-utils/lib/rendering/RenderingTestResultServer.ts

```diff
@@ -158,21 +165,56 @@ export function installMiddleware(app: express.Router, basePath: string) {
     });
 
     outputBasePath = basePath;
+    logger.log("installMiddleware output basepath:", outputBasePath);
+
     const jsonParser = bodyParser.json({ limit: 1024 * 1024 * 16 });
     app.get("/ibct-report", jsonParser, getIbctReport);
     app.post("/ibct-feedback", jsonParser, postIbctFeedback);
+    app.get("/reference-image", getReferenceImage);
```

Comment:
These changes should be in separate commits, but ok.
At least document all the featutures added in commit, because it's not only docker image.

Reply:
please verity (changed commit message and added one line of documentation... not sure if should I write something more there)